src/chocolatey/infrastructure/services/XmlService.cs

```diff
@@ -73,27 +73,17 @@ public XmlType deserialize<XmlType>(string xmlFilePath)
                                 // Check if its just a malformed document.
                                 if (ex.Message.Contains("There is an error in XML document"))
                                 {
-                                    // If so, check for a backup file and try an parse that.
-                                    if (_fileSystem.file_exists(xmlFilePath + ".backup"))
-                                    {
-                                        using (var backupStream = _fileSystem.open_file_readonly(xmlFilePath + ".backup"))
-                                        using (var backupReader = new StreamReader(backupStream))
-                                        using (var backupXmlReader = XmlReader.Create(backupReader))
-                                        {
-                                            var validConfig = (XmlType)xmlSerializer.Deserialize(backupXmlReader);
-
-                                            // If there's no errors and it's valid, go ahead and replace the bad file with the backup.
-                                            if (validConfig != null)
-                                            {
-                                                _fileSystem.copy_file(xmlFilePath + ".backup", xmlFilePath, overwriteExisting: true);
-                                            }
-
-                                            return validConfig;
-                                        }
-                                    }
-                                }

```

Comment:
No, don't change the code here, this exists for other things as well (like the config file). There is a lot of logic here based on hardening this as well, so changing this is going to bring extra scrutiny

Reply:
With the name you will give for the above, you won't need to remove this check (e.g. not `.backup`).